src/Maestro/DependencyUpdater/DependencyUpdater.cs

```diff
@@ -41,7 +41,7 @@ public sealed class DependencyUpdater : IServiceImplementation, IDependencyUpdat
             ILogger<DependencyUpdater> logger,
             BuildAssetRegistryContext context,
             IRemoteFactory factory,
-            Func<ActorId, ISubscriptionActor> subscriptionActorFactory)
+            IActorLookup<ISubscriptionActor> subscriptionActorFactory)
```

Comment:
>IActorLookup [](start = 12, length = 12)

This name is weird. It is much more a factory. It's not looking up actors, it is constructing a proxy to talk to a specific actor.

Reply:
K, I'll rename it. :-)